app/src/lib/stores/app-store.ts

```diff
@@ -2564,6 +2563,29 @@ export class AppStore extends TypedBaseStore<IAppState> {
     this._initializeCompare(repository)
   }
 
+  private async updateStashEntryCountMetric(
+    repository: Repository,
+    gitStore: GitStore
+  ) {
+    const lastStashEntryCheck = await this.repositoriesStore.getLastStashCheckDate(
+      repository
+    )
+    const dateNow = moment()
+    const threshold = dateNow.subtract(24, 'hours')
+    if (lastStashEntryCheck == null || threshold.isAfter(lastStashEntryCheck)) {
+      await this.repositoriesStore.updateLastStashCheckDate(repository)
+
+      // `lastStashEntryCheck` being equal to null means we've never checked for
```

Comment:
:art: this comment doesn't feel like it's in the right spot - where should this live?

Reply:
55a84a2